package.json

```diff
@@ -3,18 +3,6 @@
   "version": "0.1.0",
   "private": true,
   "homepage": "https://voidjmp.github.io/js-online-editor",
-  "dependencies": {
-    "@emmetio/codemirror-plugin": "^1.2.3",
-    "@testing-library/jest-dom": "^5.14.1",
-    "@testing-library/react": "^11.2.7",
-    "@testing-library/user-event": "^12.8.3",
-    "codemirror": "^5.60.0",
-    "node-sass": "^6.0.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-scripts": "4.0.3",
-    "web-vitals": "^1.1.2"
-  },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
```

Comment:
`scripts` section always should be at the top of our code. By doing this, we are making our task easier by finding which commands are available in our project.

Reply:
I agree with you